docusaurus/docs/tutorials/convert-a-frontend-datasource-to-backend.md

```diff
@@ -0,0 +1,419 @@
+---
+id: convert-a-frontend-datasource-to-backend
+title: How to convert a frontend data source plugin into a backend plugin
+sidebar_position: 10
+description: Learn how to convert a frontend data source plugin into a backend plugin
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - backend
+  - frontend
+  - datasource
+---
+
+# How to add a frontend data source to a backend component
+
+This guide shows you how to convert an existing frontend-only data source plugin into a [backend plugin](https://grafana.com/developers/plugin-tools/introduction/backend-plugins).
+
+To convert the frontend data source, we recommend scaffolding a new backend data source plugin using []`npx @grafana/create-plugin@latest`](https://grafana.com/developers/plugin-tools/). Use the following instructions to extend this foundation to copy functionality from your original plugin.
+
+## Before you begin
+
+Before you dive into the details, you should familiarize yourself with the process of creating a backend data source plugin. If you haven't done this before, you can follow our tutorial for [building a backend plugin](https://grafana.com/developers/plugin-tools/tutorials/build-a-data-source-backend-plugin).
```

Comment:
I would like to see a paragraph with a list of example use cases added near the top of this guide.

Reply:
sorry, bad wording from my side, I wanted to say a wide variety of plugins. I have rephrased that but also added a small section here for the use cases for a bakend plugin